include/evmc.h

```diff
@@ -549,6 +549,10 @@ struct evmc_instance
     /// represented by this file is in ::EVMC_ABI_VERSION.
     const int abi_version;
 
+    const char* name;
+
+    const char* version;
```

Comment:
Should have some comments (such as cannot be nullptr or empty).

Reply:
Yes, CI with doxygen has reported this.